appveyor.yml

```diff
@@ -10,7 +10,7 @@ environment:
 
 branches:
   only:
-    - master
+    - main
```

Comment:
Let's keep this change for a separate PR. This one's only for correcting link rot in documentation.